### DISCUSSION TITLE
Array Row Label is not translated

### DISCUSSION THREAD

**Discussion by GeorgeHulpoi** [payloadcms/payload#3258] [Feature Requests & Ideas]
Seems like the label is not translated. Also for a custom `RowLabel` you can't inject the `t` function for translation.

```ts
      admin: {
        components: {
          RowLabel: ({ data, index }) => {
            return data?.title || `Slide ${String(index).padStart(2, "0")}`;
          },
        },
      },
```

**Comment by JarrodMFlesch** [comment]
RowLabel has the option to be a react component. So you can create a component in another file where you import and use the translation function and then import that component in your config 😀

**Reply by GeorgeHulpoi** [reply]
@JarrodMFlesch using `useTranslation`  from `react-i18next` ?
